SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/GridBasedLBStrategy.h

```diff
@@ -48,6 +48,9 @@ class SPATIALGDK_API UGridBasedLBStrategy : public UAbstractLBStrategy
 	virtual bool RequiresHandoverData() const override { return Rows * Cols > 1; }
 
 	virtual FVector GetWorkerEntityPosition() const override;
+
+	virtual uint32 GetMinimumRequiredWorkers() const;
+	virtual void SetVirtualWorkerIds(const VirtualWorkerId& FirstVirtualWorkerId, const VirtualWorkerId& LastVirtualWorkerId);
```

Comment:
Interested to know why these are const refs? These are int32s no?

Reply:
Pure habit.

Reply:
This might also want override added. Sorry should have pointed that one out as well.

Reply:
Afaik better to pass primitives (up to double) by value over ref, but idm, I end up inadvertently passing them by ref when we have an alias for a primitive

Reply:
Theoretically we might change the type of VirtualWorkerId. lol

Reply:
Yeah I can see that argument as well. I'm not fussed either way.